src/components/common/textarea/Textarea.tsx

```diff
@@ -26,6 +27,10 @@ export default function Textarea({
 }: TextareaProps) {
   const [textAreaContentLength, setTextAreaContentLength] = useState<number>(0);
 
+  useEffect(() => {
```

Comment:
@yavuzyalcintas I added this in order to set the value of the indicator on the TextArea load event for the Edit Profile Page 
FYI